src/main/java/com/cloudogu/smeagol/wiki/domain/Page.java

```diff
@@ -0,0 +1,65 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import javax.swing.text.html.Option;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.swing.text.html.Option'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)